tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/stratifiedRules/set/SetCupStratifiedRule.scala

```diff
@@ -43,13 +44,10 @@ class SetCupStratifiedRule(rewriter: Rewriter) extends StratifiedRule[Unit] {
         throw new RewriterException("%s is not applicable".format(getClass.getSimpleName), ex)
     }
 
-  def addConstraints(scope: RewriterScope, cell: ArenaCell, auxData: Unit): Unit = {
-    val hasEdges = scope.arena.getHas(cell)
-
-    hasEdges.foreach { ptr =>
+  def addConstraints(scope: RewriterScope, cell: ArenaCell, auxData: Unit): Unit =
+    scope.arena.getHas(cell).foreach { ptr =>
       // assert here
```

Comment:
I assume this is left for later. Can we mark such places with a `// TODO` comment?

Reply:
This should be `rewriter.assert`. I assume this got messed up during the rebasing.